src/main/java/uk/gov/hmcts/reform/blobrouter/services/report/ReportService.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.reform.blobrouter.services.report;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.blobrouter.data.ReportRepository;
+import uk.gov.hmcts.reform.blobrouter.model.out.EnvelopeSummaryResponse;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.List;
+
+import static java.time.ZoneOffset.UTC;
+import static java.util.stream.Collectors.toList;
+
+@Service
+public class ReportService {
+    private final ReportRepository reportRepository;
+
+    public ReportService(ReportRepository reportRepository) {
+        this.reportRepository = reportRepository;
+    }
+
+    public List<EnvelopeSummaryResponse> getDailyReport(LocalDate date) {
+
+        var from = date.atStartOfDay().toInstant(UTC);
+        var to = date.atStartOfDay().plusDays(1).toInstant(UTC);
```

Comment:
We have to use the right time zone for London. Can't be UTC, because it may give inaccurate results.

Reply:
We use `Europe/London` in the processor

Reply:
Mmm... It is UTC in processor

Reply:
We'll need to update the processor then, too.

Reply:
@lgonczar Done. I will also raise a PR to fix this in BlobMover. For the processor I assume we need discussion and a separate Jira task